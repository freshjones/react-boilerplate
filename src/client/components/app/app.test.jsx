import React from 'react'
import renderer from 'react-test-renderer'
import App from './app'

describe('App (Snapshot)', () => {
  it('App renders Hello World!', () => {
    const component = renderer.create(<App />)
    const json = component.toJSON()
    expect(json).toMatchSnapshot()
  })
})
