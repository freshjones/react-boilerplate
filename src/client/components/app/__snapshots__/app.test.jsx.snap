// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App (Snapshot) App renders Hello World! 1`] = `
<div
  className="flex items-center justify-center"
  id="app"
>
  <h1
    className="regular"
  >
    Hello World!
  </h1>
</div>
`;
